src/components/clusterWizard/HostsTable.tsx

```diff
@@ -153,31 +156,39 @@ const HostsTable: React.FC<HostsTableProps> = ({
   const onHostEnable = React.useCallback(
     (event: React.MouseEvent, rowIndex: number, rowData: IRowData) => {
       const hostId = rowData.extraData.id;
-      enableClusterHost(clusterId, hostId).catch((err) => {
-        console.error('Failed to enable host in cluster: ', err);
-        addAlert({
-          key: `enable-${hostId}`,
-          variant: AlertVariant.warning,
-          text: `Failed to enable host ${hostId}`,
+      enableClusterHost(clusterId, hostId)
+        .then(() => {
+          dispatch(forceReload());
+        })
+        .catch((err) => {
+          console.error('Failed to enable host in cluster: ', err);
```

Comment:
it would be good to use handleApiError here

Reply:
good point, done